packages/grid/backend/alembic/versions/2021-09-20_916812f40fb4.py

```diff
@@ -0,0 +1,25 @@
+"""empty message
```

Comment:
Can you think of a name for this migration @IonesioJunior?

Reply:
Fixed by 2100b512ffb188374e1d883cd2f359586182596b